src/admin-menus/class-sniff-page.php

```diff
@@ -14,7 +14,7 @@
 use Theme_Sniffer\Exception\No_Themes_Present;
 
 /**
-* Theme sniffer page menu class
+* Theme sniffer page menu class.
```

Comment:
Can you revert this in all docblocks in all the files? The summaries should not end in a full stop.

Reply:
I'll